Flick/Controllers/SettingsViewController.swift

```diff
@@ -78,6 +79,7 @@ class SettingsViewController: UIViewController {
         UserDefaults.standard.removeObject(forKey: Constants.UserDefaults.authorizationToken)
         UserDefaults.standard.removeObject(forKey: Constants.UserDefaults.userId)
         URLCache.shared.removeAllCachedResponses()
+        ImageCache.default.clearMemoryCache()
```

Comment:
from what i remember, they use both memory and disk cache, so we can also clear  ImageCache's disk cache?